### PR TITLE
Muparser shared

### PR DESCRIPTION
**Pull request by bugbrains**
add muparserx 
activate muparser shared build

### CODE REVIEW

File: src/muparserx.mk [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+echo "TODO: write update for muparserx"
+endef
+
+define $(PKG)_BUILD
+    mkdir '$(1).build'
+    cd '$(1).build' && cmake \
+    -DCMAKE_TOOLCHAIN_FILE='$(CMAKE_TOOLCHAIN_FILE)' \
+    -DBUILD_SHARED_LIBS=$(if $(BUILD_STATIC),FALSE,TRUE) \
+    -DCMAKE_CXX_FLAGS='-std=c++11' \
+    '$(1)'
```
Comment (TimothyGu) [inline src/muparserx.mk]: Indentation

File: src/muparserx.mk [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 3.0.5
+$(PKG)_CHECKSUM := d8f646aac4099973b70ae84e1c7cb411abb6fe0b
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $($(PKG)_SUBDIR).tar.gz
+$(PKG)_URL      := http://sourceforge.net/projects/mxedeps/files/$($(PKG)_FILE)
+$(PKG)_DEPS     := gcc
+
+define $(PKG)_UPDATE
+echo "TODO: write update for muparserx"
```
Comment (TimothyGu) [inline src/muparserx.mk]: Indentation

File: src/muparserx.mk [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := muparserx
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 3.0.5
+$(PKG)_CHECKSUM := d8f646aac4099973b70ae84e1c7cb411abb6fe0b
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $($(PKG)_SUBDIR).tar.gz
+$(PKG)_URL      := http://sourceforge.net/projects/mxedeps/files/$($(PKG)_FILE)
```
Comment (TimothyGu) [inline src/muparserx.mk]: No, no mirrors.

File: src/muparserx.mk (hunk 1 above)
Comment (bugbrains) [inline src/muparserx.mk]: fixed

File: src/muparserx.mk (hunk 3 above)
Comment (bugbrains) [inline src/muparserx.mk]: fixed

Comment (tonytheodore) [inline src/muparserx.mk]: What was fixed? `http://sourceforge.net/projects/mxedeps/files` is not a valid primary source. Or secondary for that matter if it isn't part of a mirror or cdn.

**Comment by tonytheodore** [comment]
I've added this from the [official Github source](https://github.com/beltoforion/muparserx)

See:
https://github.com/mxe/mxe/commit/e8d882a99ea75d06a66a71da1db6bebbcdffb79f
https://github.com/mxe/mxe/commit/3465e4db8a5004569c0bfe9f58b82a3c34234eb0
https://github.com/mxe/mxe/commit/f939f4a9b785277a24556584d302aa251e9e932c

**Pull request closed without merging**